gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -102,8 +102,8 @@ export default function NxGlobalSidebarPage() {
                 <NxTable.Cell>
                   The path to an image placed in the top left corner which displays branding and product name.
                   <NxWarningAlert>
-                    Note: the image should not be larger than 145px wide and 32px tall. It should
-                    display either a logo (apx 32px x 32px) and the product name, or just the product name.
+                    Note: the image should not be larger than 158px wide and 43px tall. It should
+                    display either a logo (maximum 43px x 43px) and the product name, or just the product name.
                   </NxWarningAlert>
```

Comment:
I'm wondering if this should be adjusted to no longer be a warning/instruction.  We now have what we need to support larger images and our code simply resizes them.  Additionally, these instructions are ambiguous for sizeless images such as SVGs.  So maybe replace this warning with something saying that the image will be rendered at a size no greater than 158x43.  I think we can also drop the bit about the 43x43 logo.

Reply:
>I'm wondering if this should be adjusted to no longer be a warning/instruction. We now have what we need to support larger images and our code simply resizes them. Additionally, these instructions are ambiguous for sizeless images such as SVGs. So maybe replace this warning with something saying that the image will be rendered at a size no greater than 158x43. I think we can also drop the bit about the 43x43 logo.

Yep, that makes sense! [2ec3b0a](https://github.com/sonatype/sonatype-react-shared-components/pull/1078/commits/2ec3b0a1cc54504504e02aeb94151a7fa64c8755)